styles/books/neuroscience/book.scss

```diff
@@ -124,13 +124,48 @@ $ChapterIntroType: side;
   ),
 ));
 
+// Tables
+@include add_settings((
+  NoCellBordersTable: (
+    _selectors: ('.os-table.os-no-cellborder-container'),
+    'TableContainer:::display': table,
+    'TableContainer:::margin-left': auto,
+    'TableContainer:::margin-right': auto,
+    'Table:::border-collapse': collapse,
+    'Table:::margin-top': v-spacing(1),
+    'Table:::margin-bottom': 0,
+    'Table:::width': 100%,
+    'TableBottomBorderAfterTable:::border-top-color': (_ref: 'colorMap:::neuroscienceTableColor'),
+    'CaptionTitleLabel:::color': (_ref: 'colorMap:::neuroscienceTableColor'),
+    'CaptionTitleLabel:::font-family': (_ref: 'typography:::titleOption1Font'),
+    'CaptionTitle:::font-family': (_ref: 'typography:::titleOption1Font'),
+    'CaptionText:::font-family': (_ref: 'typography:::sansFont'),
+    'CaptionNumber:::color': (_ref: 'colorMap:::neuroscienceTableColor'),
+    'CaptionNumber:::font-family': (_ref: 'typography:::titleOption1Font'),
+    'NoBordersTableHeadCell:::font-weight': 700,
+    'TableData:::padding-top': 0.16rem,
+    'TableData:::padding-bottom': 0.16rem,
+    'TableData:::padding-right': table-h-spacing(1),
+    'TableData:::padding-left': table-h-spacing(1),
+    'TableBottomBorder:::border-bottom-color': (_ref: 'colorMap:::neuroscienceTableColor'),
+  ),
+));
+
 // Book specific styles
 @include use('BookRoot', "common:::BookRoot");
 @include use('LearningObjectivesNumbered', 'carnival:::LearningObjectivesNumberedShape');
 
 // TOC
 @include use('TOC', 'carnival:::NoUnitToc');
 
+// EOB
+@include add_settings((
+  AppendixGlossary: (
+    _selectors: (".appendix > .column-container"),
+    'H3AppendixGlossaryTitle:::color': (_ref: "colorMap:::neuroscienceHeaderColor"),
+    'AppendixGlossaryContainer:::column-count': 2,
+  ),
+));
```

Comment:
I see there are some changes from another PR. Did you base this branch on one from https://github.com/openstax/ce-styles/pull/358? There are two options now, either you merge 358 first, then update this branch with main and make sure there are no other changes only those for tables, or you can make another commit here in which you will remove all unnecessary changes (L: 161-168 and L: 222-223), but remember to rebuild all the styles, so also output will be updated.